src/arsciexp.py

```diff
@@ -11,6 +11,7 @@
 import arload
 import arcomb
 import arflux
+import armasters
```

Comment:
In general, I think the `use masters` implementation works well throughout this code. The way master frames are now stored deprecates a previous implementation I had (your approach is better), where a user can specify the name of a master calibration frame. This was done with the command:

`reduce usebias master_filename.fits`

At the moment, the new `use_masters` implementation means that masters are always used or never used. Perhaps we can slightly modify the old `reduce usebias` implementation to have another acceptable argument, something like:

`reduce usebias master`

then, we could use the conditional:

`if self._argflag['masters']['use'] or self._argflag['masters']['usebias']:`

and a similar argument for the other types of master calibration frames.

Whaddya think?